### PR TITLE
About me setup

### PR DESCRIPTION
**Pull request by aszulc97**
Hey babes, I just moved some code around - code that refers to both sites is now in files outside components (css and ts).
I did also some basic styling for about me page. Approve plx xoxo

### CODE REVIEW

**Review by Much4cho — APPROVED** [review]

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -4,13 +4,13 @@ import { AboutMeComponent } from './about-me/about-me.component';
 import { MainPageComponent } from './main-page/main-page.component';
 
 const routes: Routes = [
-  { path: 'about-me', component: AboutMeComponent},
+  { path: 'aboutme', component: AboutMeComponent },
```
Comment (Much4cho) [inline src/app/app-routing.module.ts:7]: why change default path?

Comment (aszulc97) [reply src/app/app-routing.module.ts:7]: It's not default, it was set manually

Comment (Much4cho) [reply src/app/app-routing.module.ts:7]: either way, I would leave it with the dash. It's easier to read such url

File: src/app/about-me/about-me.component.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  flex: 2;
+  height: 100%;
+}
+
+.aboutMeText {
+  text-align: center;
+  margin: 2rem;
+}
+
+/*changes that are supposed to be in styles.css, but then they'll destroy a layout of
```
Comment (Much4cho) [inline src/app/about-me/about-me.component.css:21]: as a rule of thumb is better to not merge in commented code, unless a good reason to do otherwise

**Review by Much4cho — APPROVED** [review]

**Pull request merged**